var/spack/repos/builtin/packages/py-astropy/package.py

```diff
@@ -72,3 +71,9 @@ def build_args(self, spec, prefix):
             args.extend(['-j', str(make_jobs)])
 
         return args
+
+        if '+wcslib' in spec:
```

Comment:
The dependency on `wcslib` is not conditional, so you can remove this `if` (i.e. always do the below update).

Reply:
Oh yes, that makes sense, sorry!